index.js

```diff
@@ -29,7 +29,6 @@ const FLAGS = {
     DEBUG: 2 ** 19,
     C_DEBUG: 2 ** 20,
     DP_TRAVEL_RULE: 2 ** 21,
-    DP_BFXPAY: 2 ** 22,
```

Comment:
I suggest just to comment this line so we don't accidentally introduce something else on place of this